### ISSUE TITLE
[Feature Request] - Show progress on workspace indicator and windows

### ISSUE THREAD

**Issue by ghost** [material-shell/material-shell#231]
As a user, when an app I'm using is emitting progress status for something, ie: Downloading something from the browser, I would like to see the progress indicator on both the workspace (containing the window) and the window emitting the progress

**Comment by PapyElGringo** [comment]
I like the idea but I have no idea how to implement that. I have no knowledge of this kind of API.

**Comment by dioio-io** [comment]
Yeah it seems a bit more complicated than the previous, i will be looking through solutions, i think i've seen some extensions with this behavior
